changelog.yaml

```diff
@@ -18,7 +18,16 @@
 # Do not change the order ("NEXT" needs to be on top).
 - version: "NEXT"
   date: TBD
-  
+  changes:
+  - type: internal
+    impact: patch

```

Comment:
Are existing releases / changes not usable due to this "bug"?
Should we mark the old versions as buggy somehow?

Reply:
We might also track bugs with GitHub issues and add a field `issueNumber`?

Reply:
The old version is still supported by the latest tekton version.
It is just not allowed to mix old and new syntax.
https://github.com/tektoncd/pipeline/blob/v0.14.3/pkg/apis/pipeline/v1alpha1/task_validation.go#L65
https://github.com/tektoncd/pipeline/blob/master/pkg/apis/pipeline/v1alpha1/task_validation.go#L66